api/app/auto_spatial_advisory/fuel_stats.py

```diff
@@ -0,0 +1,65 @@
+"""
+Functions for computing fuel type stats
+"""
+from datetime import date
+from typing import List, Optional
+from wps_shared.db.models.auto_spatial_advisory import HfiClassificationThreshold, SFMSFuelType as DBSFMSFuelType
+from wps_shared.schemas.fba import (
+    AdvisoryCriticalHours,
+    ClassifiedHfiThresholdFuelTypeArea,
+    HfiThreshold,
+    SFMSFuelType
+)
+
+"""
+    <!-- Percent Curing default values from June 2021 -->
+    <!-- November 1 to July 15 - 60% -->
+    <!-- July 16 to July 30 - 70% -->
+    <!-- July 31 to August 15 - 80% -->
+    <!-- August 16 - October 31 (end of season) - 90% -->
+"""
+def get_optional_percent_curing(grass_curing_date: date, sfms_fuel_type: DBSFMSFuelType):
+    # 12 is the sfms fuel id for grass types
+    if sfms_fuel_type is None or sfms_fuel_type.fuel_type_id != 12:
+        return None
+    
+    # we can straddle a prev calendar year and the next calendar year with this date range case, nov 1, 2024 and jul 15, 2025
+    if grass_curing_date.month == 11 or grass_curing_date.month == 12:
+        if grass_curing_date >= date(grass_curing_date.year, 11, 1) and grass_curing_date <= date(grass_curing_date.year + 1, 7, 15):
```

Comment:
I'm not clear on why this nested `if` clause is necessary? If the month is November or December the percent curing will always be 60.

Reply:
The following `elif` clause should handle Jan 1 - July 15 I think.

Reply:
It's to handle the prev year -> curr year case. This test will fail without the extra conditional:

```
@pytest.mark.parametrize(
    "for_date",
    [
        (date(2024, 11, 1)), # nov 1, start of range
        (date(2024, 12, 22)), # dev 22, mid range
        (date(2025, 7, 15)), # jul 15, end of range
    ],
)
def test_60_percent_curing(for_date: date):
    assert get_optional_percent_curing(for_date, grass_fuel_type) == 60
```